linter.py

```diff
@@ -4,9 +4,6 @@
 class Phpmd(Linter):
     cmd = ('phpmd', '${file}', 'text')
     regex = (
-        # For now, do *NOT* capture 'filename' since phpmd reports 'real'
-        # paths, and Sublime and SL cannot map such paths to the original
-        # `view.file_name()`.
         r'(.+):(?P<line>\d+)\s*(?P<message>.+)$'
```

Comment:
We do support `(?P<filename>)`

Reply:
Oh, I missed that bit in the docs. It says...

> You can capture filename if the linter emits for multiple files at once. Very often linters don’t do that and then it’s best to omit capturing filename as SublimeLinter will figure out the correct value for you and you don’t run into problems with unnamed buffers or possibly symlinked files.

... so that still means capturing filenames here is useless, this linter reports one file at a time.

> Do you know how sublimelinter.com could point to the "latest" docs, it still refers the "stable" docs?

I have no idea how that stuff (Sphinx I think) works, and can't find anything in any obvious config files. 🤷🏻

Reply:
I thought this is a configuration of the domain sublimelinter.com thing.  Do you own this domain still btw?  Or is it public domain and in the org?

Reply:
And yeah we can remove the comment if you like, but the commit message is bonkers as you tell us `SL does not even support capturing filename...?` which is not really a standard commit message from your handbook.

Reply:
> Do you own this domain still btw

No, one of the old maintainers has it, I need to look that up.

Reply:
That's also a configuration of readthedocs.org, "our" domain redirects to it.  But I don't think I have the credentials for it.

Reply:
I think I found it.

<img width="845" alt="Screenshot 2023-08-04 at 14 19 57" src="https://github.com/SublimeLinter/SublimeLinter-phpmd/assets/2543659/fd4ed910-da92-4680-bc9e-6ad80f6d2b60">

I can also add your email address to that account and you should be able to access it too.

Reply:
So "stable" should match the last release that's also available on package control.

Reply:
Yes stable is the release and latest is master.  But for SublimeLinter we don't release a new version if the docs change.  How did you login to readthedocs, with GitHub credentials?

Reply:
my email address and a password

Reply:
I don't seem to be able to add your email address to the account. I believe doing so, and sharing the password with you, would allow you to access the account as well. Maybe you also have another email address we could use?

<img width="508" alt="Screenshot 2023-08-04 at 17 24 24" src="https://github.com/SublimeLinter/SublimeLinter-phpmd/assets/2543659/41a5bbad-c408-4b65-949f-ecfa5bc272ed">

Reply:
You can't add me as a maintainer?  E.g. for mockito (which I maintain) I have 

![image](https://github.com/SublimeLinter/SublimeLinter-phpmd/assets/8558/c362bf5e-e69d-4c0c-9e64-d83fc7e27783)

Reply:
Sent you an invite!

![Uploading Screenshot 2023-08-24 at 14.41.58.png…]()

Reply:
Got it!  Thanks.